curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/inform-with-the-paragraph-element.md

```diff
@@ -9,15 +9,15 @@ dashedName: inform-with-the-paragraph-element
 
 # --description--
 
-`p` elements are the preferred element for paragraph text on websites. `p` is short for "paragraph".
+`p` elements are the preferred element for paragraph text on websites. `p` is short for `paragraph`.
```

Comment:
```suggestion
`p` elements are the preferred element for paragraph text on websites. `p` is short for "paragraph".
```
Chances are this will be translated as 
> `p` is short for the English term "paragraph"

Something like that.